netbox/dcim/graphql/types.py

```diff
@@ -434,7 +450,9 @@ class InterfaceType(IPAddressesMixin, ModularComponentType, CabledObjectMixin, P
 @strawberry_django.type(
     models.InterfaceTemplate,
     fields='__all__',
-    filters=InterfaceTemplateFilter
+    filters=InterfaceTemplateFilter,
+    pagination=True
+
```

Comment:
We should remove this.

Reply:
I wasn't sure what you meant on this one?  I think pagination should be here?

Reply:
Sorry, I meant the blank line (line 455).